### PR TITLE
Update Debian and Fedor packages as requested by itzg(Geoff)

### LINKED ISSUE #77: Align supported agent installation platforms
## What do you want us to know about?  

**Report a documentation issue**
- [ ] Bug (misspellings, grammatical problems, broken link, format issues)
- [ ] Missing information
- [x] Incorrect information
- [ ] Feedback or suggestions
- [ ] Other

**Product**
- [ ] Product bug
- [ ] Product feedback

**Tell us more about your issue**

We realized we haven't been doing a good job of coordinating new platforms and deprecated platforms for the agent installations described at

https://developer.rackspace.com/docs/rackspace-monitoring/v1/getting-started/install-configure/

Here are a set of changes that will get the documentation back in alignment with our supported platforms:

Distribution | Remove | Add | Notes
---|---|---|---
Debian | wheezy, testing, unstable | buster, jessie | Resulting list will be buster, stretch, jessie
Redhat | 5 | |
CentOS | 5 | |
Fedora | 16 - 21 | 22 - 30 |

Thanks!

### PR DESCRIPTION
**Pull request by cyrichardson**
Closes #77  - last updates requested.

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 14 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-04c09850e/docs/rackspace-monitoring/v1/). valid for 14 days

**Comment by rackernexus** [comment]
Build 15 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-c9725a4b3/docs/rackspace-monitoring/v1/). valid for 14 days

**Comment by cyrichardson** [comment]
@itzg All done pending your approval.

File: api-docs/getting-started/install-configure.rst [hunk 1]
```diff
@@ -557,17 +593,23 @@ is running a Red Hat, Fedora, or CentOS operating system.
    +--------------------------------------+--------------------------------------+
    | Red Hat Enterprise Linux 7, x86-64   | redhat-7.asc                         |
    +--------------------------------------+--------------------------------------+
-   | Fedora 16, x86-64                    | fedora-16.asc                        |
+   | Fedora 22, x86-64                    | fedora-22.asc                        |
```
Comment (itzg) [inline api-docs/getting-started/install-configure.rst:596]: Thanks for catching these.

**Review by itzg — APPROVED** [review]
Thanks!

**Pull request merged**